src/commands/dockerInstallerBase.ts

```diff
@@ -0,0 +1,67 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Change file name to `DockerInstallerBase.ts` (classes get capitalized file names, at least in this repo).